test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1

```diff
@@ -0,0 +1,2086 @@
+﻿
+if (-not (Get-Module TestRemoting -ErrorAction SilentlyContinue))
+{
+    $root = git rev-parse --show-toplevel
+    $remotingModule = Join-Path $root test/powershell/Common/TestRemoting.psm1
+    Import-Module $remotingModule
+}
+
+Describe "Implicit remoting and CIM cmdlets with AllSigned and Restricted policy" -tags "Feature" {
+
+    BeforeAll {
+        
+        # Skip test for non-windows machines
+        $skipTest = !$IsWindows
+
+        if ($skipTest) { return }
```

Comment:
If we just return here then I don't think we get a "skip" count when running tests.  Isn't the pattern to use an It block with -skip so skipped tests will be recorded?